contrib/capitalone_dataprofiler_expectations/package_info.yml

```diff
@@ -14,8 +14,13 @@
 
 general:
   package_name: CapitalOne DataProfiler Expectations
-  icon: ~ # May be a relative path to `assets/` or a URL
-  description: Expectations using CapitalOne's DataProfiler library
+  icon: https://1000logos.net/wp-content/uploads/2018/11/Capital-One-Logo.png
```

Comment:
minor nit (and in the interest of time we could probably leave for later), but without knowing much about this site, we may want to download the image and host it ourselves.